primitives/teerex/src/lib.rs

```diff
@@ -97,6 +98,44 @@ pub enum SgxStatus {
 	ConfigurationNeeded,
 }
 
+impl From<TcbStatus> for SgxStatus {
+	fn from(tcb_status: TcbStatus) -> Self {
+		match tcb_status {
+			TcbStatus::UpToDate => Self::Ok,
+			TcbStatus::SWHardeningNeeded => Self::Ok,
```

Comment:
I am a bit wary of the information loss here, we simply swallow the `SWHardeningNeeded` without printing it anywhere, or am I mistaken?

Reply:
this seems to be what IAS does as well, as SWhardeningneeded doesn't exist there

we do get it in the logs:
https://github.com/integritee-network/pallets/blob/d24538dea033f3ab130892929dadc06076c41e47/primitives/teerex/src/lib.rs#L374-L375

Reply:
Alright, then I am satisfied